please/please.py

```diff
@@ -11,172 +11,192 @@
 from rich.table import Table
 
-# INITIALIZE PACKAGES
 app = typer.Typer()
 console = Console()
 
 
-def center(text: str):
+def center(text: str) -> str:
     return text.center(shutil.get_terminal_size().columns)
 
 
-def center_print(text: str, style: str = None):
+def center_print(text: str, style: str = None) -> None:
     return console.print(Align.center(text), style=style)
 
 
-def center_print_wrap(text: str, style: str):
-    return console.print(Align.center(text, style=style, width=shutil.get_terminal_size().columns//2))
+def center_print_wrap(text: str, style: str) -> None:
+    return console.print(
+        Align.center(text, style=style, width=shutil.get_terminal_size().columns // 2)
+    )
 
 
-def write_config(data):
-    with open(os.path.join(config_path, "config.json"), 'w') as of:
-        of.write(json.dumps(data, indent=2))  # 'indent' formats json properly
+def write_config(data: dict) -> None:
+    with open(os.path.join(config_path, "config.json"), "w") as of:
+        of.write(json.dumps(data, indent=2))
 
 
-def get_time_of_day(x):
-    if (x > 4) and (x <= 12):
-        return 'Morning'
-    elif (x > 12) and (x <= 16):
-        return 'Afternoon'
-    elif (x > 16) and (x <= 20):
-        return 'Evening'
-    elif (x > 20) and (x <= 24) or (x <= 4):
-        return'Night'
+# function is never called
 
+# def get_time_of_day(x:int) -> str:
+#     if 4 < x <= 12:
+#         return "Morning"
+#     elif 12 < x <= 16:
+#         return "Afternoon"
+#     elif 16 < x <= 20:
+#         return "Evening"
+#     elif (20 < x <= 24) or (x <= 4):
+#         return "Night"
 
-def all_tasks_done():
-    completed_all_tasks = True
-    for task in config["tasks"]:
-        if task["done"] == False:
-            completed_all_tasks = False
-    return completed_all_tasks
 
+def all_tasks_done() -> bool:
+    return all(task["done"] for task in config["tasks"])
 
-@ app.command(short_help="Change name without resetting data")
-def callme(name: str):
+
+@app.command(short_help="Change name without resetting data")
+def callme(name: str) -> None:
     config["user_name"] = name
     write_config(config)
     center_print("\nThanks for letting me know your name!\n", "black on green")
 
 
-@ app.command(short_help='Add a Task')
-def add(task: str):
-    new_task = {
-        "name": task,
-        "done": False
-    }
+@app.command(short_help="Add a Task")
+def add(task: str) -> None:
+    new_task = {"name": task, "done": False}
     config["tasks"].append(new_task)
     write_config(config)
-    center_print(f"Added \"{task}\" to the list",
-                 "cyan1 on purple3")
+    center_print(f'Added "{task}" to the list', "cyan1 on purple3")
     print_tasks(config["tasks"])
 
 
-@ app.command(short_help='Deletes a Task')
-def delete(index: int):
+@app.command(short_help="Deletes a Task")
+def delete(index: int) -> None:
     index = index - 1
-    if index >= len(config["tasks"]):
+    if len(config["tasks"]) == 0:
```

Comment:
The conditions aren't mutually exclusive, it's better not to use if/elif here (mutually exclusive conditions should be handled in their own if/elif statements), so would have to be reverted unfortunately.